site/content/docs/concepts/labels.md

```diff
@@ -24,6 +24,30 @@ LABEL project_root "~/go/src/"
 
 Will bind mount the host directory `$project_root/<org>/<repo>` to `~/go/src/<repo>` in the container.
 
+### Run on Open Labels
+
+You can run a command in your sail container after it starts by specifying
+the `on_open` label. If you'd like to run multiple commands on launch, we
+recommend using a `.sh` file as your `on_open` label, as you cannot provide
+multiple `on_open` statements.
+
+The `on_open` label is run detached inside of `/bin/sh` as soon as the
```

Comment:
I think we should use bash. The difference is small, but bash is certainly used more.